src/index.js

```diff
@@ -18,7 +23,159 @@ import { months } from './data.js'
   leftArrow.addEventListener('click', () => { scrollItems('left') })
   rightArrow.addEventListener('click', () => { scrollItems('right') })
   document.getElementById('select-icon').addEventListener('change', updateMiddleCitiesScrollButtonVisibility)
+  changeBottomByContinents('down-arrow')
```

Comment:
Do we need to execute both this at first? Should calling anyone by default be enough?